src/testModel.ts

```diff
@@ -62,7 +62,7 @@ export class TestModel extends DisposableBase {
   private _vscode: vscodeTypes.VSCode;
   readonly config: TestConfig;
   private _projects = new Map<string, TestProject>();
-  private _playwrightTest: PlaywrightTestCLI | PlaywrightTestServer;
+  private _playwrightTest: PlaywrightTestServer | PlaywrightTestCLI;
```

Comment:
This improves the DX, when we command click on `_playwrightTest.Foo` it now lands in the TestServer implementation rather than in the CLI impl.